ingestion/src/metadata/profiler/metrics/composed/null_ratio.py

```diff
@@ -48,10 +48,12 @@ def fn(self, res: Dict[str, Any]) -> Optional[float]:
         Safely compute null ratio based on the profiler
         results of other Metrics
         """
+        import pandas as pd
+
         res_count = res.get(Count.name())
         res_null = res.get(NullCount.name())
 
         if res_count is not None and res_null is not None:
-            return res_null / (res_null + res_count)
-
+            result = res_null / (res_null + res_count)
```

Comment:
any potential division by 0 error here?

Reply:
Had come across NaN related issue in certain scenarios, that's why the check around pd.isna